exemplar/src/main/kotlin/com/squareup/exemplar/HelloWebAction.kt

```diff
@@ -1,9 +1,6 @@
 package com.squareup.exemplar
 
-import misk.web.Get
-import misk.web.PathParam
-import misk.web.RequestHeaders
-import misk.web.ResponseContentType
+import misk.web.*
```

Comment:
update your IntelliJ configs to disable wildcard imports
https://github.com/square/java-code-styles/

Reply:
Yeah, that's set, I'm not sure why it's doing that.